ui/raidboss/data/05-shb/raid/e6n.js

```diff
@@ -216,7 +216,7 @@
         'Eruption': 'Eruption',
         'Downburst': 'Fallböe',
         'Conflag Strike': 'Feuersbrunst',
-        'Call of the Inferno': 'Flimmernde Hitze',
+        'Call Of The Inferno': 'Flimmernde Hitze',
         'Blaze': 'Flamme',
         'Air Bump': 'Aufsteigende Böe',
```

Comment:
`Hands of Flame, and Hands of Hell` are in the `txt` file so you need the translators. I haven't seen `'Call Of The Inferno'`, but isn't she used as a triger?

Reply:
Oh. If you look through the DE block, you will see that each of the abilities I mentioned appears twice, once in alphabetical order and once at the end of the block. I was just suggesting that you could remove the duplicates at the end of the block. I wasn't commenting specifically on Call of the Inferno, it just looks that way because Github's markup doesn't work well.

Reply:
Ok, it's corrected.